src/main/java/com/ticketbookingapp/adapters/booking/BookingRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.ticketbookingapp.adapters.booking;
+
+import com.ticketbookingapp.domain.booking.Booking;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+interface BookingRepository extends CrudRepository<Booking, Long> {
```

Comment:
Czy użycie CrudRepository to dobry pomysł? Jaką zasadę z SOLID łamie CrudRepository?